python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -112,6 +112,11 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
     if missing_cred:
         raise Exception("Failed due to missing credentials %s" % str(missing_cred))
 
+    # Make sure that all the config options are really set.
+    # Some from plugins may not have taken during startup
```

Comment:
It's not controversial, I'm just very keen to make sure this is understood as I've found in the past the config system rarely does what I'd expect it to in some cases.
If this is due to the plugins then OK, at least it's the loading of the plugins causing it to happen.
Aside from the config there's little here that isn't in the main code so we're not that different thankfully